backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/handlingunits/HUEditorRow.java

```diff
@@ -264,6 +271,21 @@ private HUEditorRow(@NonNull final Builder builder)
 		{
 			attributesSupplier = Optional.empty();
 		}
+
+		if (hasAttributes() && attributesSupplier.get().get().hasAttribute(AttributeConstants.ATTR_SerialNo))
+		{
+			serialNo = attributesSupplier.get().get().getValueAsString(AttributeConstants.ATTR_SerialNo);
+		}
+		else
+		{
+			serialNo = "A";
```

Comment:
does that mean we print an `A` if an HU has no serial number?

Reply:
on a first look, yes it should.
however, in my local testing i never got the "A" displayed. The column was always empty or with a valid string. Now i wonder why.

Reply:
Fixed